charts/merlin/templates/database-password-secret.yaml

```diff
@@ -1,23 +0,0 @@
-{{- if not .Values.postgresql.enabled }}
-apiVersion: v1
-kind: Secret
```

Comment:
How is database password being stored when it's using "external"?

Reply:
It will be created using the same way as the previous one. using the `postgresqlPassword` parameter